app/store/models/entities.ts

```diff
@@ -107,9 +107,9 @@ export default interface Entities {
   [EntityType.Surveys]: Record<ID, Survey>;
   [EntityType.Tags]: Record<ID, UnknownTag>;
   [EntityType.Users]: Record<ID, UnknownUser>;
-  // [EntityType.FollowersCompany]: Record<ID, unknown>; // AFAIK unused
-  // [EntityType.FollowersUser]: Record<ID, unknown>; // AFAIK unused
-  // [EntityType.FollowersEvent]: Record<ID, unknown>; // AFAIK unused
+  [EntityType.FollowersCompany]: Record<ID, unknown>; // AFAIK unused
+  [EntityType.FollowersUser]: Record<ID, unknown>; // AFAIK unused
+  [EntityType.FollowersEvent]: Record<ID, unknown>; // AFAIK unused
```

Comment:
Unused?

Reply:
yes, but commenting them out fucks with the types... It is a problem for a another PR